osgtest/tests/special_install.py

```diff
@@ -69,9 +69,16 @@ def test_03_update_osg_release(self):
         command = ['rpm', '-e', '--nodeps', 'osg-release']
         core.check_system(command, 'Erase osg-release')
 
-        self.assert_(re.match('\d+\.\d+', core.options.updaterelease), "Unrecognized updaterelease format")
-        rpm_url = 'https://repo.opensciencegrid.org/osg/' + core.options.updaterelease + '/osg-' + \
-                  core.options.updaterelease + '-el' + str(core.el_release()) + '-release-latest.rpm'
+        update_release = core.options.updaterelease
+        self.assert_(re.match('\d+[\.]\d+$', update_release), "Unrecognized updaterelease format")
```

Comment:
```suggestion
        self.assert_(re.match('\d+[.]?\d+$', update_release), "Unrecognized updaterelease format")
```
otherwise it's not gonna match again.

Reply:
alternatively
```suggestion
        self.assert_(re.match('[0-9.]+$', update_release), "Unrecognized updaterelease format")
```